ntbs-service/Pages/Shared/_AlertsAndActions.cshtml

```diff
@@ -84,24 +84,43 @@
                     </print-button>
                 }
 
-                @if (Model.Notification.NotificationStatus is NotificationStatus.Notified or NotificationStatus.Closed 
-                     && Model.PermissionLevel is PermissionLevel.Edit)
+                @if (Model.PermissionLevel is PermissionLevel.Edit)
                 {
-                    var transferRequestHref = @RouteHelper.GetNotificationPath(Model.NotificationId, NotificationSubPaths.TransferRequest);
-                    <a id='transfer-button' class="nhsuk-button ntbsuk-button--primary" role="button" href="@transferRequestHref">
-                        @if (@Model.Notification.TransferRequestPending)
+                    if (Model.Notification.NotificationStatus is NotificationStatus.Notified or NotificationStatus.Closed)
```

Comment:
Why are you preferring a double `if` block over the `&&` that we previously had?